docs/en/getting-started/docker/docker-compose.yml

```diff
@@ -1,13 +1,13 @@
-version: '2.2'
+version: "2.2"
```

Comment:
IIUC, this has been optional for a while and the current version is 3.8. Do we use any features below that require us to stay in 2.2 ? Can we change to latest or remove this ? @bytebilly might know

Reply:
We are rewriting the `docker-compose.yml` file entirely, so we don't know yet.
Since this is for getting started, we should not assume users have the latest `docker-compose` and it would be nice to make it compatible with the lowest version possible.

It's better to make it explicit, since some of the keywords have been removed in version 3 but they still work if you specify version 2 in the file.